webapp/src/routes/Profile/ProfilePageDonor.js

```diff
@@ -97,19 +97,17 @@ const useStyles = makeStyles((theme) => ({
     flexDirection: 'column',
     alignItems: 'center',
   },
-  editBtn: {
-    borderRadius: '50px',
-    backgroundColor: '#ba0d0d',
-    width: "70%",
-    fontSize: '14px',
-    fontWeight: 500,
-    fontStretch: 'normal',
-    fontStyle: 'normal',
-    lineHeight: 1.14,
-    letterSpacing: '1px',
+  editButton: {
+    borderRadius: 50,
+    height: 60,
+    padding: 20,
```

Comment:
Shouldn't this be transferred to the theme?

Reply:
I meant that maybe we can use global styles in the theme instead of writing them right here. In that way the styles are standardised. But no problem, we can omit that and continue for now